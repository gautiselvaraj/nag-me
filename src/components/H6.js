import React from 'react';
import styled from 'styled-components';

const Heading = styled.h6`
  font-size: 0.95rem;
  margin-top: 0.15rem;
  margin-bottom: 0.15rem;
`;

export default ({ children, ...otherProps }) => (
  <Heading {...otherProps}>{children}</Heading>
);
