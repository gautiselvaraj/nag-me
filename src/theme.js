export default {
  main:           '#214154',
  mainDarken:     '#083148',
  mainLighten:    '#215a7b',
  black:          '#111111',
  white:          '#f6f6f6',
  greyLightest:   '#efefef',
  greyLighter:    '#dddddd',
  greyLight:      '#a5a5a5',
  grey:           '#898989',
  greyDark:       '#777777',
  greyDarker:     '#595959',
  greyDarkest:    '#222',
}
