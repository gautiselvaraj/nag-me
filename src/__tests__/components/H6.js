import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import H6 from '../../components/H6';
import theme from '../../theme';
import 'jest-styled-components';

describe('<H5 />', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<H6 theme={theme}>Heading 6</H6>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should have correct text & element', () => {
    const h6 = mount(<H6 theme={theme}>Heading 6</H6>);
    expect(h6.text()).toBe('Heading 6');
    expect(h6.find('h6').length).toBe(1);
  });
});
