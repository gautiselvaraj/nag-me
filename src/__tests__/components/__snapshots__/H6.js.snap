// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<H5 /> should match snapshot 1`] = `
.c0 {
  font-size: 0.95rem;
  margin-top: 0.15rem;
  margin-bottom: 0.15rem;
}

<h6
  className="c0"
>
  Heading 6
</h6>
`;
